packages/vtex-io-logger/tests/core/instrumentation/routes/get-binding-for-route.spec.ts

```diff
@@ -33,6 +42,7 @@ describe("getBindingsForRoute", () => {
       workspace: "testWorkspace",
       traceId: "testTraceId",
       type: "route",
+      appId: process.env.VTEX_APP_ID,
```

Comment:
Just to make the test more verbose, considering that we don't know the function gets from process.env

```suggestion
      appId: "vendor.app-name@0.0.1",
```